main.py

```diff
@@ -490,19 +493,43 @@ def delTrans():
 
 def editBook():
     def addBook():
+        mycursor =  mycon.cursor()
+        mycursor.execute(f"INSERT INTO books VALUES {addBookIDE.get()},{addBookNameE.get()},{addBookAuthE.get()},{addBookGenreE.get()},{addBookPubE.get()},{addBookYoPE.get()},{addBookPriceE.get()},{addBookNewE.get()},{addBookSecE.get()}")
```

Comment:
since we have AUTO_INCREMENT, would it be possible for the application itself to display the bookid? instead of requiring the user to type it?